metadata.rb

```diff
@@ -23,8 +23,8 @@
 depends 'golang'
 depends 'firewall', '~> 2.0'
 depends 'poise', '~> 2.2'
+depends 'poise-archive', '~> 1.3'
 depends 'poise-service', '~> 1.4'
-depends 'rubyzip', '~> 1.3'
```

Comment:
You've removed `rubyzip`, but `consul_installation_binary` resource is still using `zipfile` resource from it. So, it's broken and ChefSpec test fails.